webpack.config.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+const webpack = require('webpack');
+const wds_port = 4050;
+
+const PATHS = {
+    src: path.join(__dirname, 'src'),
+    js: path.join(__dirname, 'src/js'),
+    build: path.join(__dirname, 'dist'),
+    devServer: path.join(__dirname, 'dev-server'),
+    demo: path.join(__dirname, 'demo')
+};
+
+console.log('Webpack running with environment: ', process.env.NODE_ENV)
+
+//default environment to production if not set
+if (!process.env.NODE_ENV) {
+  process.env.NODE_ENV = 'production';
+}
+
+//environment specific config
+let includes = [PATHS.js]
+let devtool = false
+let output_path = PATHS.build
+let entrypoint = PATHS.js + '/index.js'
+let plugins = [new webpack.EnvironmentPlugin(['NODE_ENV'])]
+
+if (process.env.NODE_ENV == 'development') {
```

Comment:
i would recommend cleaning all this up by using two different webpack configs: one for development and one for production.

no hurry though if this is working now.